### PR TITLE
taxonomy: changes for week 01

### PR DESCRIPTION
**Pull request by aleene**
### What
Some changes for week 1

### CODE REVIEW

File: taxonomies/nutrients.txt [hunk 1]
```diff
@@ -3713,7 +3713,7 @@ dv_value:en: 1.7
 unit:en: mg
 
 zz:vitamin-pp
-en:Vitamin B3, Vitamin PP (Niacin), Vitamin B3, Vitamin PP, Niacin
+en:Vitamin B3 (Niacin), Vitamin PP (Niacin), Vitamin B3, Vitamin PP, Niacin
```
Comment (moon-rabbitOFF) [inline taxonomies/nutrients.txt:3716]: I thought we don't had things between parenthesis in the taxonomy?

Comment (aleene) [reply taxonomies/nutrients.txt:3716]: I use it as a solution to fill the pop-up list. Sometimes a package says Niacin and sometimes B3. And I never remember which one goes with what. So now I get always the correct entry proposed. I also started using this approach for the categories, as the suggestions are based on the first letters only. This makes targeting very difficult in dutch (adjectives before the noun).

Comment (alexgarel) [reply taxonomies/nutrients.txt:3727]: I confirm, it's ok to have a descriptive label as first entry if you repeat each entry separately.

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**